### ISSUE TITLE
No source map support?

### ISSUE THREAD

**Issue by binarykitchen** [juliangruber/tape-run#40]
With the browserify command, all gets bundled and in case of an error, it does not point to the correct line number of the source code. Is there a trick to make this happen?

**Comment by syarul** [comment]
I don't think this is the issue with tape-run, it's the issue of ```tape``` which don't [ as yet?] support source mapping regardless of compiler (browerify/webpack/rollup)
